ui/app/components/secure-variable-form.hbs

```diff
@@ -39,17 +39,31 @@
         .
       </p>
     {{/if}}
+    <SecureVariableForm::NamespaceFilter 
+      @data={{hash 
+        disabled=(not @model.isNew)
+        selection=this.variableNamespace
+        namespaceOptions=this.namespaceOptions
+      }}
+      @fns={{hash
+        onSelect=this.setNamespace
+        setNamespaceOptions=this.setNamespaceOptions
+      }}
+    />
```

Comment:
Noticing a style / positioning error when receiving a "path already exists" error
![image](https://user-images.githubusercontent.com/713991/180019399-0392afb6-0d64-4c67-ae57-4558b947ccf9.png)

Reply:
The issue here is that, we started this work before this change was merged in and now because we're expecting a new child, we need to rethink the design of the whole grid.

I also believe that the `relative` position attribute will need to be rethought for this.